### ISSUE TITLE
Github actions for deployment

### ISSUE THREAD

**Issue by bbollen23** [visdesignlab/loonar#72]
We need to have a deployment structure to do the following when we merge into main:

- [x] 1. Create local and standard images
- [x] 2. Deploy new docker container to server
- [x] 3. Generate New docs
- [x] 4. Update images in docs (we might possibly have a releases page to show various version of Loon)

EDIT 09/10/24:

Added github actions to create standard and local images. When created, push two images for each of local and standard loon. We push a "latest" version so that the user can just call a standard simple command without specifying the tag, and a versioned one that is an exact copy of "latest" so that we can still have all the versions in docker hub. Works like this:

Merge to main 1:
-> builds loon.
-> tags with "latest"
-> pushes latest one
-> tags again with "version number"
-> pushes versioned one

Merge to main 2:
-> builds loon
-> tags with "latest"
-> pushes latest one which will overwrite the "old" latest version
-> tags again with " new version number"
-> pushes versioned one

After the "Merge to main 2", we will have 3 versions: v1, v2, and latest, where latest will point to a copy of v2. 

No need for "4". Dockerhub already has a very specific tags page we can direct to instead which doesn't change. The user can see two links (per image) in the docs: 
1. One pointing to the overview page which will have the docker command to pull the latest version.
2. The tags page where they can see all versions

EDIT 09/12/24:

Created new "loonar-deployment" repository: https://github.com/visdesignlab/loonar-deployment

This has a flask server that we can deploy so that we can listen for requests from Github Actions on port 8443. Waiting on IT ticket to open up port 8443.

EDIT 09/13/24:

Versioning docusaurus is outside the scope of this since we currently have no needs for versions.
